doc/about.rst

```diff
@@ -0,0 +1,47 @@
+.. _about:
+
+======================
+About ``mybinder.org``
+======================
+
+This page explains some of the teams and organizations behind mybinder.org.
+
+What technology runs mybinder.org?
+==================================
+
+The technology behind mybinder.org is primarily composed of three open-source projects:
+
+* `JupyterHub <https://z2jh.jupyter.org>`_, which manages cloud infrastructure for user instances
+* `repo2docker <repo2docker.readthedocs.io>`_, which builds Docker images from GitHub repositories
+* `BinderHub <binderhub.readthedocs.io>`_, which orchestrates the above two projects and
+  provides the Binder interface.
+
+Each of these are open and inclusively-governed projects. Currently, these are all officially
+hosted as a part of Project Jupyter, an open project that creates open tools for data science
+infrastructure and interactive computation. The mybinder.org team is
+heavily involved in each, though mybinder.org does not need to have a controlling
+stake in any of them.
+
+Who runs mybinder.org?
+======================
+
+mybinder.org is a single deployment of the BinderHub technology. It is run as a free, public
+service to the scientific and educational community. It also serves as a way for the Binder
+project to test the "cutting edge" of Binder and demonstrate the technology.
+
+Currently, the mybinder.org team is comprised of the `Binder Project team <https://jupyterhub-team-compass.readthedocs.io/en/latest/team.html#binder-team>`_.
+However, as time moves on we expect that these two teams will start to diverge as more
+institutions deploy BinderHub for their own purposes, and want to help govern the direction
+of the project.
```

Comment:
I'd remove this paragraph from this document and put it in a document on how to join the Binder project. If you deploy your own BinderHub you don't end up involved with mybinder.org. You can however be involved in running mybinder.org without deploying your own BinderHub. If you deploy your own Hub you probably want to become part of the Binder project to help set direction in which the team goes. But maybe you are happy to be another contributor to the software projects (BinderHub and repo2docker) without being involved in the Binder project.

Reply:
cool, removed it in the latest push. WDYT? I'll hold off on adding a "how you can get involved" page to keep this PR scoped more cleanly